.github/workflows/screenshots.yml

```diff
@@ -45,4 +45,4 @@ jobs:
         run: npm start --workspace govuk-frontend-review &
 
       - name: Send screenshots to Percy
-        run: npx percy exec -- npm run test:screenshots --workspace govuk-frontend-review
+        run: npx --workspace govuk-frontend-review percy exec -- npm run test:screenshots
```

Comment:
Rather than run `npx percy` at project level, we can also run it from the workspace